src/ripple/RippleTools.ts

```diff
@@ -91,14 +106,22 @@ export class RippleTools implements EdgeCurrencyTools {
     }
   }
 
-  async importPrivateKey(privateKey: string): Promise<{ rippleKey: string }> {
-    privateKey = privateKey.replace(/\s/g, '')
+  async importPrivateKey(input: string): Promise<JsonObject> {
     try {
-      // Try decoding seed
-      decodeSeed(privateKey)
-
-      // If that worked, return the key:
-      return { rippleKey: privateKey }
+      if (validateMnemonic(input)) {
+        // Try creating wallet
+        Wallet.fromMnemonic(input, { mnemonicEncoding: 'bip39' })
+        return {
+          rippleMnemonic: input
+        }
+      } else {
+        const privateKey = input.replace(/\s/g, '')
```

Comment:
Yes, this line of code actually does something useful. We should fix the Stellar import to work the same.